### ISSUE TITLE
Need a way to handle snapshots when they "vanish"

### ISSUE THREAD

**Issue by jacksodj** [Huddle/PhantomCSS#160]
currently a snapshot of a control that disappears generates the following error:
FAIL Cannot upload nonexistent file: ./test/visual/dev-www/results/home-marquee-mobile.diff.png
 #    type: uncaughtError
 #    error: Cannot upload nonexistent file: ./test/visual/dev-www/results/home-marquee-mobile.diff.png
 #           phantomjs://platform/casper.js:831:101
 #           map@[native code]
 #           _forEach@phantomjs://platform/casper.js:829:26
 #           _forEach@[native code]
 #           forEach@[native code]
 #           fillForm@phantomjs://platform/casper.js:824:38
 #           fillSelectors@phantomjs://platform/casper.js:908:25
 #           asyncCompare@phantomjs://platform/phantomcss.js:314:22
 #           phantomjs://platform/phantomcss.js:413:16
 #           runStep@phantomjs://platform/casper.js:1595:31
 #           checkStep@phantomjs://platform/casper.js:404:28
 #    stack: not provided

it would be handy to have it generate a transparent png of the word "MISSING" for the diff file when the baseline is present

then the failure file would be the base image with the word missing in really large print.

**Comment by jamescryer** [comment]
Yeah, it could handle the error differently, it would need to check for the elements existence before creating the screenshot. It would be a behavioural change so the user would need to enable it via a new option parameter (for backward compatibility).

I'll be happy to review a Pull Request for this feature.
